internal/pkg/api/schema.go

```diff
@@ -175,3 +175,47 @@ type StatusResponse struct {
 	Status  string                 `json:"status"`
 	Version *StatusResponseVersion `json:"version,omitempty"`
 }
+
+type FileInfo struct {
+	ActionID string     `json:"action_id"`
+	AgentID  string     `json:"agent_id"`
+	Source   string     `json:"src"`
+	File     FileData   `json:"file"`
+	Contents []FileData `json:"contents"`
+	Event    struct {
+		ID string `json:"id"`
+	} `json:"event"`
+	Host struct {
+		Hostname string `json:"hostname"`
+	} `json:"host"`
+}
+
+type FileData struct {
+	Size        int64  `json:"size"`
+	Name        string `json:"name"`
+	Extension   string `json:"ext"`
+	Mime        string `json:"mime_type"`
+	Compression string `json:"Compression"`
+	Hash        struct {
+		SHA256 string `json:"sha256"`
+		MD5    string `json:"md5"`
+	}
+	Accessed    string   `json:"accessed"`
+	Attributes  []string `json:"attributes"`
+	Created     string   `json:"created"`
+	CTime       string   `json:"ctime"`
+	Device      string   `json:"device"`
+	Directory   string   `json:"directory"`
+	DriveLetter string   `json:"drive_letter"`
+	Ext         string   `json:"extension"`
```

Comment:
Line 196 has
```
Extension string `json:"ext"`
```
It's a a bit confusing

Reply:
This is all going to get removed soon. After some back and forth, mapping will be set to dynamic so fleet server has no need to know or decode whatever the JSON structure will be. So most of these fields are getting removed, just need to decide on a nice way for us to inject one or two properties into an otherwise nearly-anything-goes payload, and check a few minimum-required fields.